dev/Repeater/TestUI/Samples/FlexboxAlignContentPage.xaml

```diff
@@ -0,0 +1,135 @@
+﻿<Page
+    x:Class="MUXControlsTestApp.Samples.FlexboxAlignContentPage"
+    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+    xmlns:local="using:MUXControlsTestApp.Samples"
+    xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+    xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+    xmlns:controls="using:Microsoft.UI.Xaml.Controls"
+    mc:Ignorable="d"
+    Background="{ThemeResource ApplicationPageBackgroundThemeBrush}">
+
+    <StackPanel>
```

Comment:
Consider adding API tests - see flowlayout tests for examples  on how to write layout tests. [https://github.com/microsoft/microsoft-ui-xaml/blob/main/dev/Repeater/APITests/FlowLayoutTests.cs](https://github.com/microsoft/microsoft-ui-xaml/blob/main/dev/Repeater/APITests/FlowLayoutTests.cs)